app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -1177,7 +1177,7 @@ private void adjustSeekbarsThumbs() {
         // Set Thumb 1 drawable to an orange marker and value at the max possible Pokemon level at the current
         // trainer level
         expandedLevelSeekbarBackground.getThumb(0).setThumb(getDrawableC(R.drawable
-                .orange_seekbar_thumb_marker));
+                .orange_marker));
```

Comment:
This code is inside a `Service`, not an `AppCompatActivity`, so IIUC what @thearaks explained, it's going to fail (right)? Or does that issue not apply to `getDrawable`?